crates/bevy_text/src/error.rs

```diff
@@ -1,10 +1,11 @@
-use ab_glyph::GlyphId;
 use thiserror::Error;
 
 #[derive(Debug, PartialEq, Eq, Error)]
 pub enum TextError {
     #[error("font not found")]
     NoSuchFont,
     #[error("failed to add glyph to newly-created atlas {0:?}")]
-    FailedToAddGlyph(GlyphId),
+    FailedToAddGlyph(u16),
+    #[error("font system mutex could not be acquired or is poisoned")]
```

Comment:
What does "poisoned" mean in this context?

Reply:
The mutex itself has been poisoned, which I think happens when the thread that held it panicked.

Reply:
> The mutex itself has been poisoned
Is this terminology that is commonly used in this domain? I'm unfamiliar with it and wanting to learn.

Reply:
Seems like it is: 
https://blog.logrocket.com/understanding-handling-rust-mutex-poisoning/

Reply:
Yeah that's correct. I only have a shallow understanding of it and I just used the existing terminology. Handling the mutex(es) was one of the things I asked for help with in my original PR.

Reply:
Yep, this is the correct terminology to use in this PR.